config/go.d/couchbase.conf

```
@@ -0,0 +1,96 @@
+# netdata go.d.plugin configuration for couchbase
+#
+# This file is in YAML format. Generally the format is:
+#
+# name: value
+#
+# There are 2 sections:
+#  - GLOBAL
+#  - JOBS
+#
+#
+# [ GLOBAL ]
+# These variables set the defaults for all JOBs, however each JOB may define its own, overriding the defaults.
+#
+# The GLOBAL section format:
+# param1: value1
+# param2: value2
+#
+# Currently supported global parameters:
+#  - update_every
+#    Data collection frequency in seconds. Default: 10.
+#
+#  - autodetection_retry
+#    Re-check interval in seconds. Attempts to start the job are made once every interval.
+#    Zero means not to schedule re-check. Default: 0.
+#
+#  - priority
+#    Priority is the relative priority of the charts as rendered on the web page,
+#    lower numbers make the charts appear before the ones with higher numbers. Default: 70000.
+#
+#
+# [ JOBS ]
+# JOBS allow you to collect values from multiple sources.
+# Each source will have its own set of charts.
+#
+# IMPORTANT:
+#  - Parameter 'name' is mandatory.
+#  - Jobs with the same name are mutually exclusive. Only one of them will be allowed running at any time.
+#
+# This allows autodetection to try several alternatives and pick the one that works.
+# Any number of jobs is supported.
+#
+# The JOBS section format:
+#
+# jobs:
+#   - name: job1
+#     param1: value1
+#     param2: value2
+#
+#   - name: job2
+#     param1: value1
+#     param2: value2
+#
+#   - name: job2
+#     param1: value1
+#
+#
+# [ List of JOB specific parameters ]:
+#  - url
+#    Server URL.
+#    Syntax:
+#      url: http://localhost:8091
+#
+#  - username
+#    Username for basic HTTP authentication.
+#    Syntax:
+#      username: admin
+#
+#  - password
+#    Password for basic HTTP authentication.
+#    Syntax:
+#      password: 123456
+#
+#
+# [ JOB defaults ]:
+#  url: http://127.0.0.1:8091
+#  timeout: 10
+#
+#
+# [ JOB mandatory parameters ]:
+#  - name
+#  - url
+#
+# ------------------------------------------------MODULE-CONFIGURATION--------------------------------------------------
+# [ GLOBAL ]
+# update_every: 10
+# autodetection_retry: 0
+# priority: 70000
+#
+#
+# [ JOBS ]
+# jobs:
+#   - name: local
+#     url: http://127.0.0.1:8091
+#     username: admin
+#     password: password
```

```suggestion
jobs:
   - name: local
     url: http://127.0.0.1:8091
     username: admin
     password: password
```

let's have this job uncommented. couchbase instance running on localhost => we collect data from it.